code/modules/reagents/reagent_containers/chem_pack.dm

```diff
@@ -0,0 +1,48 @@
+/obj/item/reagent_containers/chem_pack

```

Comment:
you can just put a beaker in the iv drop does this do anything important

Reply:
The chem packs  can be renamed with a pen and are spillproof + harder to sabotage when sealed.
In addition they are less likely to get stolen by someone who needs a beaker. 

Also the doctors intuitively know that it is intended for the IV drip when they see it in the fridge or on a table.  

I think having proper IV bags adds to the medbay atmosphere as well.